.tmuxinator.yml

```diff
@@ -0,0 +1,4 @@
+name: stackbuilders/hapistrano
+windows:
+  - editor: nvim .
+  - terminal: null
```

Comment:
Cool! I didn't know that we could have a tmuxinator config per project.

Reply:
@CristhianMotoche my bad, I was not planning to commit this file 😂 I could leave it here if that is OK with you, however, not everyone uses `nvim`

Reply:
That's true. Let's remove it then.